src/Symfony/Component/Mailer/Transport/Smtp/EsmtpTransportFactory.php

```diff
@@ -25,8 +25,9 @@ public function create(Dsn $dsn): TransportInterface
         $tls = 'smtps' === $dsn->getScheme() ? true : null;
         $port = $dsn->getPort(0);
         $host = $dsn->getHost();
+        $verifyPeer = $dsn->getOption('verify-peer', true);
```

Comment:
I think we should use an underscore here, that'd be consistent with other DSNs (eg from Cache).